routes/index.js

```diff
@@ -1,9 +1,11 @@
 const moviesRoute = require("./movies");
 const usersRoute = require("./users");
+const recsRoute = require("./recommendations")
 
 const routesSetup = (app) => {
-  app.use("/api/movies", moviesRoute);
```

Comment:
@HalfMillennium the "/api" part in front of each route is necessary. can you please update to the following?

```javascript
  app.use("/api/movies", moviesRoute);
  app.use("/api/users", usersRoute);
  app.use("/api/recs", recsRoute)
```